app/src/main/java/com/rlab/sejima/MainActivity.java

```diff
@@ -1,13 +1,71 @@
 package com.rlab.sejima;
 
-import android.support.v7.app.AppCompatActivity;
+import android.graphics.Color;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.AppCompatEditText;
+import android.util.Log;
+import android.view.View;
+
+import com.rlab.sejima.features.MUTextField;
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends AppCompatActivity implements View.OnClickListener, MUTextField.MUTextFieldListener {
+
+//    private boolean editable = true;
+//    private boolean autocorrect = true;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        findViewById(R.id.btn_1).setOnClickListener(this);
+        findViewById(R.id.btn_2).setOnClickListener(this);
+        findViewById(R.id.btn_3).setOnClickListener(this);
+        findViewById(R.id.btn_4).setOnClickListener(this);
+    }
+
+    @Override
+    public void onClick(View v) {
+        switch(v.getId()){
+            case R.id.btn_1:
+                Log.e(getClass().getCanonicalName(), "add listener");
+//                ((MUTextField) findViewById(R.id.mutf1)).setSecure(true);
```

Comment:
Is the comment for test only?

Reply:
Yes, I removed it. Was just to test features at runtime.